lhotse/bin/modes/workflows.py

```diff
@@ -55,6 +56,36 @@ def workflows():
 @click.option(
     "-d", "--device", default="cpu", help="Device on which to run the inference."
 )
+@click.option(
+    "--faster-whisper",
+    is_flag=True,
+    default=True,
+    help="If True, use faster-whisper's implementation based on CTranslate2.",
+)
+@click.option(
+    "--faster-whisper-use-vad",
+    is_flag=True,
+    default=True,
+    help="If True, use faster-whisper's built-in voice activity detection (SileroVAD)."
+    "Note: This requires onnxruntime to be installed.",
+)
+@click.option(
+    "--faster-whisper-add-alignments",
+    is_flag=True,
+    default=False,
+    help="If True, add word alignments using timestamps obtained using the cross-attention"
+    "pattern and dynamic time warping (Note: Less accurate than forced alignment).",
+)
+@click.option(
+    "--faster-whisper-compute-type",
+    default="float16",
```

Comment:
```suggestion
    default="auto",
```

Otherwise it won't work on (some?) CPUs.